addons/core/addon/components/pending-confirmations/index.js

```diff
@@ -14,7 +14,8 @@ import { action } from '@ember/object';
  */
 export default class PendingConfirmationsComponent extends Component {
   // =services
-
+  // confirm service use within template
+  // eslint-disable-next-line ember/no-unused-services
```

Comment:
Judging by the rule's [limitations](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-unused-services.md), how frequent do we have unused services vs using the service only in the template or other scenarios? I'm wondering if we'll have more exceptions than valid catches in the end